src/main/kotlin/com/terraformation/backend/admin/AdminDocumentProducerController.kt

```diff
@@ -61,6 +63,39 @@ class AdminDocumentProducerController(
     return documentProducerAdminHome()
   }
 
+  @Operation(
+      summary = "Upload a variable manifest.",
+      description = "The uploaded file must be in CSV format.",
+      externalDocs =
+          ExternalDocumentation(
+              description = "Current Google sheet",
+              url =
+                  "https://docs.google.com/spreadsheets/d/16ybKFseSwFubxZ_i6ENxsF2XT0RqajhdumefJOoHQo8/edit#gid=1050189377"))
```

Comment:
We should make sure the permissions on this document are locked down tight since terraware-server is a public repo.

Reply:
It is currently only accessible to people with a Terraformation email

Reply:
Looks that way to me too; we'll just need to make sure we're careful with sharing that doc from now on since we don't have security-by-obscurity as an added hurdle.